excel_import_export/models/xlsx_import.py

```diff
@@ -204,7 +204,7 @@ def _import_record_data(self, import_file, record, data_dict):
             out_st = out_wb.add_sheet("Sheet 1")
             xml_id = (
                 record
-                and self.get_external_id(record)
+                and self.sudo().get_external_id(record)
```

Comment:
Shouldn't we be adjusting this line https://github.com/OCA/server-tools/blob/3a0fa2a012c0db05df05454674667a19a6d33b65/excel_import_export/models/xlsx_import.py#L47 instead, adding `.sudo()` after `self`?